src/wamr/funcs.cpp

```diff
@@ -51,38 +112,13 @@ static void __faasm_write_output_wrapper(wasm_exec_env_t exec_env,
     call.set_outputdata(outBuff, outLen);
 }
 
-/**
- * Chain a function by function pointer
- */
-static int32_t __faasm_chain_ptr_wrapper(wasm_exec_env_t exec_env,
```

Comment:
These two methods were just moved somewhere else in the file to preserve alphabetical order.